src/resolvers/__tests__/collection.test.js

```diff
@@ -710,6 +710,112 @@ describe('Collection', () => {
         })
       })
     })
+
+    describe('collection', () => {
+      describe('when calling revisions returns a revision tombstone', () => {
```

Comment:
suggest:
`describe('when retrieving revisions that contain tombstones')
test('includes them in the results')
`